db/migrate/20240102155112_create_standings_and_user_standings.rb

```diff
@@ -0,0 +1,28 @@
+class CreateStandingsAndUserStandings < ActiveRecord::Migration[7.0]
+  def change
+    create_table :standings do |t|
+      t.string :name
+      t.string :color
+      t.text :description
+      t.boolean :default
+      t.datetime :archived_at
```

Comment:
@harigopal 

I am adding a partial index for `standings.default` because currently we have queries targeting only default: true and default: true records will be lesser when compared to default: false.

```rb
add_index :standings, :default, where: "default = true"
```

In the future, if we decide to show the archived standings on the admin side, then also that page will not be accessed much, and I think a partial index is more beneficial than a full index